src/commands/sort_ro.json

```diff
@@ -37,7 +37,8 @@
             },
             {
                 "flags": [
```

Comment:
spoke to Guy, indeed the reason the key-spec for STORE is both `unknown` and `INCOMPLETE` is that during the search for the STORE keyword, we we could find the BY and GET pattern (which could be the word "store") and then we'll return the wrong result .
i.e. for that reason i'ts not just marked as INCOMPLETE like other specs, it avoids even attempting to find that key (hence the `unknown`.

what Guy recommended was to add a 3rd spec of `unknown` type, with `INCOMPLETE` for the GET and BY args, rather than add the INCOMPLETE flag to the first key-spec (for the key we're sorting)

Reply:
p.s. pity we can't add comments in the json files, maybe we need to add a description or comment field per keyspec